src/vxworks/mod.rs

```diff
@@ -431,6 +415,16 @@ s_no_extra_traits! {
         pub __ss_pad2  : [::c_char; _SS_PAD2SIZE],
     }
 
+    pub union sa_u_t {
```

Comment:
The vxworks/mod.rs should be guarded in the lib/mod.rs file with a `libc_union` gate - otherwise the older rust build jobs will try to compile these and fail.

Reply:
But then the type would be undefined, so the older rust build jobs would still fail?

Reply:
Ah sorry, my bad! We only try to compile these targets with nightly in the build.rs! The older toolchains don't know these targets at all, so that would always fail.